src/etc/inc/auth.inc

```diff
@@ -593,8 +610,18 @@ function local_user_del($user) {
 
 function local_user_set_password(&$user, $password) {
 
-	$user['password'] = crypt($password);
-	$user['md5-hash'] = md5($password);
+	$user['password'] = "REMOVED";
+	$user['md5-hash'] = "REMOVED";
+	$user['bcrypt-hash'] = password_hash($password,PASSWORD_BCRYPT);
+
+	// Maintain compatibility with FreeBSD
+	// https://reviews.freebsd.org/D2742
+	// change $2y$ prefix to $2b$
+
+	if ($user['bcrypt-hash'][2] == "y") {
+		$user['bcrypt-hash'][2] = "b";
+	}
```

Comment:
Yeah, but we are not implementing in 2.2.x, but anyway it needs to stay for now since the commit was never MFC'd

Please add a XXX: line in comment saying it can be removed as soon as r284483 is MFC'd